.github/actions/configure-aws-credentials/action.yaml

```diff
@@ -0,0 +1,20 @@
+name: Configure AWS Credentials
+description: Configure AWS Credentials
+
+inputs:
+  aws_region:
+    description: 'The AWS region'
+    default: 'eu-south-1'
+    required: true
+  role_to_assume:
+    description: 'The IAM role to assume'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@5fd3084fc36e372ff1fff382a39b10d03659f355 # v2.2.0
+      with:
+        role-to-assume: ${{ inputs.role_to_assume }}
+        aws-region: ${{ inputs.aws_region }}
```

Comment:
Perhaps we could [override the default lifetime of the credentials life time](https://github.com/aws-actions/configure-aws-credentials#credential-lifetime).

Reply:
I think a good value could be 15/30 min